client.go

```diff
@@ -585,9 +575,9 @@ func (c *Client) Deploy(ctx context.Context, path string) (err error) {
 		return
 	}
 
-	// Store the produced image Digest in the config
+	// Record eht Image Digest pushed.
```

Comment:
It's ridiculous for me to point out stuff like this, I know. Nit:
```suggestion
	// Record the Image Digest pushed.
```

Reply:
I prefer you do; the perfectionist in me feels less lonely.